src/dev/intel_8254_timer.cc

```diff
@@ -294,6 +294,9 @@ Intel8254Timer::Counter::CounterEvent::process()
       case SquareWave:
         setTo(counter->period);
         break;
+      case SoftwareStrobe:
+        setTo(counter->period);
+        break;
```

Comment:
```suggestion
      case SquareWave:
      case SoftwareStrobe:
        setTo(counter->period);
        break;
```

That said, I'm a bit surprised that we want the same implementation for all of these outputs. However, it also seems that `RateGen` and `SquareWave` should be different, but they aren't. So, maybe we're fine.

Reply:
The only difference is in how clock and initial count is handled.